### ISSUE TITLE
Error in AddRegValue

### ISSUE THREAD

**Issue by yfnfif** [oleg-shilo/wixsharp#80]
Hello, Oleg!
I try to add registry values from regFile.
My code is:
RegValue[] regValues = Tasks.ImportRegFile(regFile);
project.AddRegValue(regValues);

regfile is:
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\My Product]
@="%ProgramFiles%\\My Product"

After installation in registry I have "ProgramFilesFolder\My Product"

**Comment by oleg-shilo** [comment]
I don't remember form the top of my head but you may need to go with.
```
[HKEY_LOCAL_MACHINE\SOFTWARE\My Product]
@="[ProgramFilesFolder]\My Product"
```
Have a look at the generated .wxs file. It may give you a clue.

**Comment by yfnfif** [comment]
I want to get the same result as I've got via import file in Registry Editor: Data is "%ProgramFiles%\My Product". But WixSharp replases %ProgramFiles% by ProgramFilesFolder.

Generated .wxs file:
<RegistryValue Id="RegValue" Type="expandable" KeyPath="no" Value="ProgramFilesFolder\My Product" />

**Comment by oleg-shilo** [comment]
OK. I tested it and it is a defect. 
It shouldn't take much time to fix.

**Comment by oleg-shilo** [comment]
Fixed with f1e4e95.
Will be available with the very next release
